src/Analyzers/CSharp/Tests/CSharpAnalyzers.UnitTests.projitems

```diff
@@ -167,5 +169,6 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="$(MSBuildThisFileDirectory)ConvertTypeOfToNameOf\" />
+    <Folder Include="$(MSBuildThisFileDirectory)MakeStructMemberReadOnly\" />
```

Comment:
📝 This shouldn't have been added

Reply:
@tmeschter Potentially related to the other issues we're seeing with project system?

@sharwell not sure what's up.  clearly something is adding this stuff (See the line just above as well).

Reply:
Yes, to add a new item to a new folder in a shared project, you have to create the file on disk, and then add it to the project by showing all items and right clicking the item to include it in the project. If you try to create the file using the IDE, it will mess up the project file.